lti_consumer/plugin/urls.py

```diff
@@ -29,32 +20,32 @@
 
 app_name = 'lti_consumer'
 urlpatterns = [
-    url(
+    re_path(
```

Comment:
The `USAGE_ID_PATTERN` is a regex pattern and hence these paths also require re_path to function normally otherwise tests fail.